pkg/limits/flags.go

```diff
@@ -0,0 +1,123 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license
+
+package limits
+
+import (
+	"flag"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/timescale/promscale/pkg/limits/mem"
+	"github.com/timescale/promscale/pkg/util"
+)
+
+var (
+	MemoryTargetMetric = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: util.PromNamespace,
+			Name:      "memory_target_bytes",
+			Help:      "The number of bytes of memory the system will target using",
```

Comment:
Will target using? Does the kernel reserves itself a constant memory or variable memory?

Reply:
This is used when deciding whether we can grow internal datastructures (like cache) to stay below the memory target